.buildkite/scripts/common.sh

```diff
@@ -51,6 +51,18 @@ with_go() {
     export PATH="${PATH}:$(go env GOPATH):$(go env GOPATH)/bin"
 }
 
+with_msft_go() {
+    echo "Setting up microsoft/go"
+    create_workspace
+    check_platform_architeture
+    MSFT_DOWNLOAD_URL=https://aka.ms/golang/release/latest/go$(cat .go-version)-1.${platform_type}-${arch_type}.tar.gz
+    retry 5 $(curl -sL -o - $MSFT_DOWNLOAD_URL | tar -xz -f - -C ${WORKSPACE})
+    export PATH="${PATH}:${WORKSPACE}/go/bin"
+    go version
+    which go
+    export PATH="${PATH}:$(go env GOPATH):$(go env GOPATH)/bin"
```

Comment:
Why is `$(go env GOPATH)` needed here? AFAIK there's no executable of interest there...
```suggestion
    export PATH="${PATH}:$(go env GOPATH)/bin"
```